providers/fips/fipsprov.c

```diff
@@ -634,10 +666,37 @@ int OSSL_provider_init(const OSSL_CORE_HANDLE *handle,
 
     fgbl->handle = handle;
 
+    /*
+     * We did initial set up of selftest_params in a local copy, because we
+     * could not create fgbl until c_CRYPTO_zalloc was defined in the loop
+     * above.
+     */
+    fgbl->selftest_params = selftest_params;
+
+    fgbl->selftest_params.libctx = libctx;
+
+    set_self_test_cb(fgbl);
+
+    if (!fips_get_params_from_core(fgbl)) {
+        /* Error already raised */
+        return 0;
+    }
+    /*
+     * Disable the conditional error check if is disabled in the fips config
```

Comment:
@slontis, why are we keeping these?

Reply:
What do you mean?
There are are few configurable items that were requested. This was one of them.

Reply:
Turning them off kind of invalidates FIPS.

Reply:
Yep.
Needs to be added to security policy.

Reply:
We should make a list of "things to go in the security policy"